wcivf/urls.py

```diff
@@ -33,16 +33,14 @@
             name="django.contrib.sitemaps.views.sitemap",
         ),
         url(r"^robots\.txt", include("robots.urls")),
-        url(r"^email/", include("dc_signup_form.urls")),
+        url(r"^mailing_list/", include("mailing_list.urls", "dc_signup_form")),
     ]
     + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
     + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
 )
 
 
 if settings.DEBUG:
-    import debug_toolbar
+    from dc_utils.urls import dc_utils_testing_patterns
```

Comment:
File to add this in, but not sure if we should remove Django debug toolbar - I find it really useful in local dev! So to include both we could do:
```
if settings.DEBUG:
    import debug_toolbar
    from dc_utils.urls import dc_utils_testing_patterns
    
    urlpatterns = [
        url(r"^__debug__/", include(debug_toolbar.urls))
    ] + dc_utils_testing_patterns + urlpatterns
```

Reply:
Can we take a closer look at this? It causes a major change to the home page.

Reply:
Super useful, thanks!